### PR TITLE
Mark all restarted (cluster) jobs in UI

### PR DESCRIPTION
**Pull request by mitiao**
See: https://progress.opensuse.org/issues/37453

Work instance:
[/tests](http://10.67.19.14/tests)
[group results overview](http://10.67.19.14/tests/overview?distri=sle&version=15&build=665.2&groupid=4)

### CODE REVIEW

**Comment by coolo** [comment]
@os-autoinst/tools-team please review

File: lib/OpenQA/WebAPI/Controller/Admin/Workers.pm [hunk 1]
```diff
@@ -87,6 +87,7 @@ sub previous_jobs_ajax {
                 push(
                     @data,
                     {
+                        DT_RowId     => "job_" . $job->id,
```
Comment (foursixnine) [inline lib/OpenQA/WebAPI/Controller/Admin/Workers.pm]: I wonder where is it used...

Comment (coolo) [reply lib/OpenQA/WebAPI/Controller/Admin/Workers.pm]: in datatables driven by ajax - google for DT_RowId

File: t/ui/26-jobs_restart.t [hunk 2]
```diff
@@ -0,0 +1,260 @@
+# Copyright (C) 2016 SUSE Linux GmbH
```
Comment (mudler) [inline t/ui/26-jobs_restart.t]: 2018 :smiley:

Comment (mitiao) [reply t/ui/26-jobs_restart.t]: correct the year :)

File: lib/OpenQA/WebAPI/Controller/Admin/Workers.pm (hunk 1 above)
Comment (foursixnine) [reply lib/OpenQA/WebAPI/Controller/Admin/Workers.pm]: Guessed so :)

**Pull request merged**